scout/server/blueprints/cases/controllers.py

```diff
@@ -287,8 +287,8 @@ def coverage_report_contents(store, institute_obj, case_obj, base_url):
     # extract default panel names and default genes from case_obj and add them to the post request object
     distinct_genes = set()
     panel_names = []
-    for panel_info in case_obj.get('panels', []):
-        if not panel_info.get('is_default'):
+    for panel_info in case_obj.get('panels', [ ]):
+        if not panel_info.get('is_default') or panel_info.get('is_default') is False:
```

Comment:
Sorry, but isn't ```is False``` already one of the cases tested by ```not```, along with ```ìs None```etc?

Reply:
I think you are right, because if the key doesn't exist it's also false.  This is the panel situation for case in #1287 :
![image](https://user-images.githubusercontent.com/28093618/63431310-dd289c80-c41e-11e9-8c59-0c1df1e264a2.png)

I have to think more..